spec/support/model_reconstruction.rb

```diff
@@ -10,7 +10,15 @@ def reset_class class_name
 
     klass.reset_column_information
     klass.connection_pool.clear_table_cache!(klass.table_name) if klass.connection_pool.respond_to?(:clear_table_cache!)
-    klass.connection.schema_cache.clear_table_cache!(klass.table_name) if klass.connection.respond_to?(:schema_cache)
+
+    if klass.connection.respond_to?(:schema_cache)
+      if `bundle exec rails -v`.include?('4.2')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.